### DISCUSSION TITLE
How to access graphql interface directly?

### DISCUSSION THREAD

**Discussion by paymog** [hasura/graphql-engine#10072] [Question] [answered]
I'd like to access the graphql interface of hasura directly (without going through the hasura admin dashboard). In the admin dashboard I see the following

<img width="1728" alt="image" src="https://github.com/hasura/graphql-engine/assets/1986950/4f7aaef4-0991-463b-8ac6-1fdc6f21777e">

When I try to navigate to that url I get the following error message

```
{"errors":[{"message":"PersistedQueryNotSupported","extensions":{"path":"$","code":"not-supported"}}]}
```

<img width="927" alt="image" src="https://github.com/hasura/graphql-engine/assets/1986950/45168190-ea23-4b1a-8a7f-21a0c6c8b5ee">

I've tried clearing my browser cache and doing a hard reload but that doesn't seem to help. 

I'm running `hasura/graphql-engine:v2.36.0` in docker with the following env vars

```
      HASURA_GRAPHQL_DATABASE_URL: <postgres uri>
      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
      HASURA_GRAPHQL_DEV_MODE: "true"
      HASURA_GRAPHQL_PG_CONNECTIONS: 2
```

How can I properly configure hasura so that the graphql interface is accessible on it's own without having to go through the admin dashboard?

**Comment by manasag** [comment, marked as answer]
https://localhost:8080/v1/graphql is a POST endpoint which serves the GraphQL API. Since you are accessing using GET (by trying to access the endpoint directly on the browser), you are encountering the error. 
The interface you are referring to (it's a modified [GraphiQL](https://github.com/graphql/graphiql) IDE web module) is hosted at https://localhost:8080, and is embedded in the Console. Since [GraphiQL](https://github.com/graphql/graphiql) is open sourced, you can try to run it yourself separately, and point to https://localhost:8080/v1/graphql for your purpose.

**Reply by paymog** [reply]
Ah gotcha, as a quick followup, is there a way to disable the various aggregation/statistical queries? Ideally I'd like to just have `CheckoutSession` and `Customer` and none of the extra possible queries.

<img width="347" alt="image" src="https://github.com/hasura/graphql-engine/assets/1986950/18c7d49f-8af3-4e37-b39a-8e826cb75a8b">

I don't have aggregation enabled for the client that's being used to access my hosted GraphiQL

<img width="1216" alt="image" src="https://github.com/hasura/graphql-engine/assets/1986950/fd6582e9-09e2-4ec1-9a39-1186588d39a0">
